lxd/daemon.go

```diff
@@ -899,6 +899,7 @@ func (d *Daemon) Init() error {
 			caPool.AddCert(ca)
 			tlsConfig.RootCAs = caPool
 			tlsConfig.ClientCAs = caPool
+			tlsConfig.InsecureSkipVerify = false
```

Comment:
Hmm, so this isn't a problem but is also completely un-needed since the default value for a boolean is false :)